rox_navigation/launch/navigation.launch.py

```diff
@@ -164,9 +180,11 @@ def generate_launch_description():
         )
     
     declare_nav2_param_file_cmd = DeclareLaunchArgument(
-            'nav2_params_file', default_value="",
-            description='Full path to the Nav2 parameters file to load.\n'
-                        '\tLeave empty to use the default file based on the robot type'
+            'nav2_params_file', default_value=os.path.join(
+                get_package_share_directory('rox_navigation'),
+                'configs',
+                'navigation_omni.yaml'),
```

Comment:
would be cool if we are able to select the diff or trike option from the launch?